com.unity.uiwidgets/Runtime/engine/UIWidgetsPanel.cs

```diff
@@ -238,6 +247,12 @@ void CollectGarbageOnDemand()
 #if !UNITY_EDITOR
             TryEnableOnDemandGC();
 #endif
+            Application.lowMemory += () => {
```

Comment:
there is an issue here:

if incremental gc is not enabled and these lines are executed, then the gc mode will be set as Disabled after the execution. It is not right.